tools/noise/dt-gen-noiseprofile

```diff
@@ -98,6 +119,11 @@ if [ "$missing_tool" = "1" ]; then
 	exit 1
 fi
 
+# Check if we have compiled dt-noiseprofile
```

Comment:
Why?

Reply:
I wanted to keep backwards compatibility, but then I will remove it.